main/util-linux/APKBUILD

```diff
@@ -11,7 +11,7 @@ esac
 
 pkgrel=0
 pkgdesc="Random collection of Linux utilities"
-url="http://git.kernel.org/cgit/utils/util-linux/util-linux.git"
+url="https://git.kernel.org/cgit/utils/util-linux/util-linux.git"
 arch="all"
 license="GPL-2.0 GPL-2.0-or-later LGPL-2.0-or-later BSD Public-Domain"
```

Comment:
Should be `LGPL-2.1-or-later`
https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/tree/README.licensing